pages/details/items/[slug].js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { getItem, getItems } from "../../../utils/data/getData";
-import ItemDetails from "../../../components/ItemDetails";
-import Layout from "../../../components/layouts/layout/Layout";
+import ItemDetails from "../../../SRC/components/ItemDetails";
+import Layout from "../../../SRC/components/layouts/layout/Layout";
```

Comment:
Why did you make a `SRC` folder? It's not wrong, I am actually curious. ;) 
Only thing i'd suggest is not making it `SCREAMING` case but simply `src`